src/Interactive.Journey/KernelExtensions.cs

```diff
@@ -17,7 +17,7 @@
 using System.Threading.Tasks;
 using Microsoft.DotNet.Interactive.CSharp;
 
-namespace Extension
+namespace Interactive.Journey
```

Comment:
Not `Microsoft.DotNet.Interactive.Journey`?

Reply:
while the package is interactive.journey i think the default namespace should be the one @jonsequitur indicates